src/css/controls/imports/autostartmute.less

```diff
@@ -0,0 +1,23 @@
+@import "../../shared-imports/vars";
+@import "../../shared-imports/icons";
```

Comment:
Why would icons be shares? What part of the icon imports apply to anything outside of controls icons?